frontend/src/composables/layout.ts

```diff
@@ -21,12 +21,12 @@ const layoutState = reactive({
 });
 
 export function useLayout() {
-  const setScale = (scale) => {
+  const setScale = (scale: number) => {
     layoutConfig.scale = scale;
-    localStorage.setItem('scale', scale);
+    localStorage.setItem('scale', scale.toString());
   };
 
-  const setActiveMenuItem = (item) => {
+  const setActiveMenuItem = (item: any) => {
```

Comment:
qui lascia stare 'any' perchè è un casino a cascata con i campi "compute" in caso faremo a tempo perso